packages/dso-toolkit/components/Componenten/form-elements/group-files.njk

```diff
@@ -10,7 +10,7 @@
     {% endfor %}
   </ul>
   {% if files | map('confidential') | includes(true) %}
-    {% render '@alert', {modifier: 'warning', sronly: 'Waarschuwing:', message:'
+    {% render '@alert', {modifier: 'warning', status: 'Waarschuwing:', message:'
```

Comment:
Ik zie dat ik deze allemaal ben vergeten om te zetten naar `<dso-alert>`, wil jij dat doen? Nergens meer `@alert`. En kan je het project nazoeken of er wellicht ergens het HTML component wordt gebruikt?

Reply:
Alle `@alert` renders zijn omgevormt tot `<dso-alert>` aanroepen